src/mito2/src/compaction/compactor.rs

```diff
@@ -162,11 +163,12 @@ pub async fn open_compaction_region(
     };
 
     Ok(CompactionRegion {
-        region_options: region_options.clone(),
         manifest_ctx,
         access_layer,
         current_version,
```

Comment:
non-blocking: Maybe we should keep fields in the same order as the fields defined in `CompactionRegion`. For example, put `manifest_ctx` and `current_version` behind `region_metadata`.